Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/IntegrationTest.cs

```diff
@@ -47,6 +49,17 @@ public void IntegrationTestOneTimeSetUp ()
       _webTestHelper = WebTestHelper.CreateFromConfiguration<CustomWebTestConfigurationFactory>();
 
       _webTestHelper.OnFixtureSetUp (MaximizeMainBrowserSession);
+      s_webApplicationRoot = new Lazy<Uri> (
+          () =>
+          {
+            var uri = new Uri (_webTestHelper.TestInfrastructureConfiguration.WebApplicationRoot);
+
+            // RM-7401: Edge loads pages slower due to repeated hostname resolution.
+            if (_webTestHelper.BrowserConfiguration.IsEdge())

```

Comment:
do we already have a follow-up task to investigate/resolve the underlying issue? To be scheduled for somewhere in 2-3 months unless there's an earlier slot available.